gcloud-java-examples/src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -142,12 +144,12 @@ public void run(Storage storage, BlobInfo... blobInfos) {
       if (blobInfos.length == 1) {
```

Comment:
Honestly, I think most of the changes in the file would look nicer if we change BlobAction and BlobsActionto have their own abstract methods and taking Blob or Bucket, Blob.... respectively.